server/services/licenceService.ts

```diff
@@ -274,26 +274,6 @@ export default class LicenceService {
     return _.head(licences)
   }
 
-  async getLicencesForApproval(user: User, prisonCaseload: string[]): Promise<LicenceSummary[]> {
```

Comment:
no need for these calls to exist here so inlined in the approval caseload service